### PR TITLE
[DOCFIX] Update CLI docs

### PR DESCRIPTION
**Pull request by calvinjia**
@gpang Could you update this page with a section on `table`?
@bradyoo Could you update this page with a section on `job`?
@chcn Could you update the page with sections on each of the `distributed*` commands `bin/alluxio fs`?

### CODE REVIEW

File: docs/en/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -104,6 +98,16 @@ in accordance to the state of the machine:
 $ ./bin/alluxio bootstrapConf <ALLUXIO_MASTER_HOSTNAME>
 ```
 
+### fs
+
+See [File System Operations](#File-System-Operations).
+
+### fsadmin
+
+The `fsadmin` command is meant for administrators of the Alluxio cluster. It provides added tools for
```
Comment (ZacBlanco) [inline docs/en/basic/Command-Line-Interface.md]: line break between sentences?

Comment (calvinjia) [reply docs/en/basic/Command-Line-Interface.md]: Updated, thanks.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please.

**Pull request merged**